src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {

```

Comment:
The interface name 'Drawable' is a verb, which is not recommended. Interface names should be adjectives or noun phrases. Consider renaming it to something like 'DrawableFigure' to better convey the intent.